models/core-java/src/main/java/org/commonjava/indy/model/core/RemoteRepository.java

```diff
@@ -365,6 +382,36 @@ public void setServerTrustPolicy( final String serverTrustPolicy )
         this.serverTrustPolicy = serverTrustPolicy;
     }
 
+    public Integer getPrefetchPriority()
+    {
+        return prefetchPriority;
+    }
+
+    public void setPrefetchPriority( Integer prefetchPriority )
+    {
+        this.prefetchPriority = prefetchPriority;
+    }
+
+    public boolean isPrefetchRescan()
+    {
+        return prefetchRescan;
+    }
+
+    public void setPrefetchRescan( boolean prefetchRescan )
+    {
+        this.prefetchRescan = prefetchRescan;
+    }
+
+    public String getPrefetchListingType()
+    {
+        return prefetchListingType;
+    }
+
+    public void setPrefetchListingType( String prefetchListingType )
+    {
+        this.prefetchListingType = prefetchListingType;
+    }
+
```

Comment:
Do we need to copy these fields in the copyOf() method?

Reply:
ok. Will do.